internal/builder/writer/human_readable_test.go

```diff
@@ -185,6 +237,8 @@ REMOTE:
 				RunImageMirrors: []string{"first/default", "second/default"},
 				Buildpacks:      buildpacks,
 				Order:           order,
+				Extensions:      buildpacks,
+				OrderExtensions: order,
```

Comment:
nit: this could be confusing to the reader as it implies that extensions can be nested, when they cannot; e.g. when reading this expected output:
```
Detection Order (Extensions):
 ├ Group #1:
 │  ├ test.top.nested@test.top.nested.version
 │  │  └ Group #1:
```
It might be better to make a separate order for extensions with its own expected output.

Reply:
Do you think I should change just the test, or do I use `dist.Order` instead of `pubbldr.DetectionOrder` to represent order for extensions? For now I have removed nesting in the test case as suggested